### DISCUSSION TITLE
PrivacyLearn.com: New site built from privacy.sexy’s scripts

### DISCUSSION THREAD

**Discussion by undergroundwires** [undergroundwires/privacy.sexy#470] [Announcements]
I’m excited to share with you a new website I’ve been working on: [PrivacyLearn.com](https://privacylearn.com/). It uses privacy.sexy’s internal script database but presents it in a static knowledge base format.

Before integrating [PrivacyLearn.com](https://privacylearn.com/) officially into the main project, I'd love to hear your thoughts and feedback on the website and how it should be integrated to privacy.sexy.

It solves following issues:

- Ability to share scripts using permalinks (#49)
- Allow access in corporates that ban `.sexy` domain (#106)
- More "tutorial-style" for non tech-savvy.
- Available for those who wants to access scripts without enabling JavaScript (#49)

Future ideas:

- More dynamism: blog, comments, voting, recently added/updated etc.
- Better integration with privacy.sexy (click-to-select in app etc)

It will be integrated to privacy.sexy in next version (0.14.0) along with `id` support (#262)

What are your thoughts?

**Comment by Fs00** [comment]
It's a very cool project! Lately I've been using privacy.sexy more as a reference than for generating scripts, so it will surely come in handy.
From what I've seen right now, the only thing that looks weird to me is that most Windows scripts run PowerShell snippets in the command prompt using `PowerShell -ExecutionPolicy Unrestricted -Command ...`, when the user could just be instructed to run the entire snippet in PowerShell directly.
But I guess this would mean making sure all privacy.sexy scripts are tailored for PowerShell, which is probably not a quick task 🙂

**Comment by ghost** [comment]
I *love* this website and look forward to seeing how it develops in the future. Thanks for the awesome privacy tool.

**Comment by undergroundwires** [comment]
Thank you for your comments and sharing your use-case @Fs00, you've been a long term contributor this project.

Update: I'm now trying to solve the SEO I destroyed somehow.

It started with around 700 indexed pages, left to 7, and Google is removing more pages every day.

<img src="https://github.com/user-attachments/assets/fe260bb3-e0b1-4950-a357-8341c7f496e0" width="600">

I've improved the UI/UX, added trust indicators (with author sections at the end of each script and an [about page](https://privacylearn.com/about)), implemented structured data, and added 'noindex' tags for all scripts missing documentation. I also modified the sitemap to exclude scripts without documentation. However, none of these changes have made any difference.

Bing's response has been even worse - they've completely banned the website and won't even index the homepage. They provided no explanation, similar to what happened with privacy.sexy:

<img src="https://github.com/user-attachments/assets/df12b0b0-599d-47b8-a697-f554254e85a6" width="500">

Yandex has indexed the important pages.

I've submitted URLs and sitemaps to those search engines and added their analytics components.

The website is nearly complete - I plan to add some more documentation pages and then let it rest for a while, as I don't have any other improvement ideas at the moment.

**Comment by ghost** [comment]
Absolutely love the new site! Quick question though. How would I run an update for the Mac if updates came out? Also, how would I know to update if updates came out? This is all if I click the blue "apply protection" button and run the script directly. 

Thank you!

**Reply by undergroundwires** [reply]
Hi. You'll need to manually visit and update from time to time as the website by its nature cannot have auto-updates/runs on the computer :(. However, this can be a good future for privacy.sexy, I'll consider this. Also incorprating your question/answer as FAQ on different pages. Thank you :)
